platform/commonUI/browse/bundle.js

```diff
@@ -226,7 +232,7 @@ define([
                         "$window"
                     ],
                     "group": "windowing",
-                    "cssClass": "icon-new-window",
+                    "cssclass": "icon-new-window",
```

Comment:
@evanohara This should be `cssClass`, not `cssclass`. Not sure how this happened, could have been rebase.